phabricator/tpl/phabricator-rc.yaml

```diff
@@ -48,7 +48,7 @@ spec:
         - name: PHABRICATOR_LASTNAME
           value: {{ .phabricatorLastName | quote }}
         - name: PHABRICATOR_HOST
-          value: {{ .phabricatorHost | quote }}
+          value: {{ default .phabricatorHostIP .phabricatorHost | quote }}
```

Comment:
If a user reserves a public ip in GKE, he/she needs to specify is as the `loadBalancerIP` in the service https://github.com/bitnami/charts/pull/94/files/7efc8ffb4c612f21eba5fdeb5a86eab62bdf1247#diff-a9050ee913dfe996ac374bddb1afba38R12. 

This makes sure that the reserved IP gets assigned to the service instead of using a dynamically allocated IP.

Since this IP is also assigned to the loadBalancer when specified, a separate field named `.phabricatorHostIP` is created.

https://github.com/bitnami/charts/pull/94/files/7efc8ffb4c612f21eba5fdeb5a86eab62bdf1247#diff-993f2e8c978b7e44a8e482f540edf7e9R51 sets the `PHABRICATOR_HOST` when the FQDN tasks precedence over the ip address incase both are specified.